cli/cli/kurtosis_gateway/server/api_container_gateway/api_container_gateway_service_server.go

```diff
@@ -31,6 +31,9 @@ type ApiContainerGatewayServiceServer struct {
 	// ServiceMap and mutex to protect it
 	mutex                               *sync.Mutex
 	userServiceNameToLocalConnectionMap map[string]*runningLocalServiceConnection
+
+	// User services port forwarding
+	connect kurtosis_core_rpc_api_bindings.Connect
```

Comment:
We store the connect attribute at the enclave level in the Gateway.  It is set in the RunStarlark functions and read in GetServices.  A cleanest way would be to add a Connect RPC method and not rely on GetServices.  The plan is to deprecate the Gateway and replace it with the Portal so we don't want to spend too much time here.